receiver/apachedruidreceiver/metadata.yaml

```
@@ -0,0 +1,2541 @@
+type: apachedruid
+
+status:
+  class: receiver
+  stability:
+    development: [metrics, logs]
+  distributions: [contrib, observiq, sumo]
+  codeowners:
+    active: [yuanlihan]
```

```suggestion
    active: [yuanlihan, atoulme]
```
I believe @atoulme should be a code owner as the sponsor of this component.